eZ/Bundle/EzPublishRestBundle/Features/ContentTypeGroup/list.feature

```diff
@@ -22,7 +22,18 @@ Feature: Read all Content Type Groups
             | dif3rent   |
             | id_nt.fier |
 
-    Scenario: Can't read any Content Type Group without an authorized user
+    Scenario: List Content Type Groups without an authorized user
```

Comment:
This should really be improved. Without an authorized user is meaningless here.

Either we use the Policy, or we use the Role. I would suggest something like "As an Anonymous User, I can list ContentTypeGroups".

"Anonymous user" matches the default user. The problem here is that we are in a bit in a special case: being the anonymous user doesn't mean much if we don't consider default data. The "Anonymous user" will differ from one project to the other. Things get easier when we actually *require* a permission to do something: "As a user with the "content/create" Policy, I can create Content". Here, we have a hard time expressing the fact that not having any Policy will grant you access to something.

I suggest we stick to "Anonymous user" until we figure out something else.

But the interesting thing here is that expressing the scenario points out that something might be wrong... (anonymous being able to list/view content types).